make/common/native/Link.gmk

```diff
@@ -132,11 +132,6 @@ define CreateDynamicLibraryOrExecutable
   ifeq ($$($1_TYPE), LIBRARY)
     # Generating a dynamic library.
     $1_EXTRA_LDFLAGS += $$(call SET_SHARED_LIBRARY_NAME,$$($1_BASENAME))
-
-    # Create loadmap on AIX. Helps in diagnosing some problems.
-    ifneq ($(COMPILER_BINDCMD_FILE_FLAG), )
```

Comment:
`spec.gmk.template` (where `COMPILER_BINDCMD_FILE_FLAG` is defined should be reverted to upstream.